source/features/align-issue-labels.tsx

```diff
@@ -1,9 +1,54 @@
 import './align-issue-labels.css';
 
+import * as pageDetect from 'github-url-detection';
+import {$, $optional} from 'select-dom/strict.js';
+
 import features from '../feature-manager.js';
+import observe from '../helpers/selector-observer.js';
 
 void features.addCssFeature(import.meta.url);
 
+/* New view */
+
+const issueSelector = '[class^="IssueRow-module__row"]';
+const badgesSelector = '[class*="trailingBadgesContainer"]';
+const mainContentSelector = '[class^="MainContent-module__inner"]';
+const descriptionSelector = '[class^="Description-module__container"]';
+const issueTypeSelector = '[class*="issueTypeTokenContainer"]';
+
+function alignBadges(issue: HTMLElement): void {
+	const badges = $(badgesSelector, issue);
+	const mainContent = $(mainContentSelector, issue);
+	const issueType = $optional(issueTypeSelector, issue);
+
+	if (badges.children.length === 0 && !issueType) {
```

Comment:
If there are no labels, we shouldn't move anything at all, right?


```suggestion
	if (badges.children.length === 0) {
```

Reply:
I like consistency, but I prefer the "suggested" version for now since it's more compact and avoids DOM changes